test/openssl/test_pkey_rsa.rb

```diff
@@ -15,8 +15,7 @@ def test_no_private_exp
 
   def test_private
     # Generated by key size and public exponent
-    key = OpenSSL::PKey::RSA.new(512, 3)
-    assert(key.private?)
+    key = OpenSSL::PKey::RSA.new(2048, 65537)
```

Comment:
```suggestion
    key = Fixtures.pkey("rsa2048")
```

There is no reason to generate a new key now.

Reply:
Fixed this on the latest rebase.